src/System.Security.Cryptography.X509Certificates/tests/ChainTests.cs

```diff
@@ -260,6 +261,132 @@ public static void BuildChain_WithCertificatePolicy_NoMatch()
         }
 
         [Fact]
+        public static void BuildChain_MicrosoftDotCom_WithSystemRootCerts()
+        {
+            // Verifies that the microsoft.com certs build with only the certificates in the root store
+            // This is a canary test for BuildChain_MicrosoftDotCom_WithRootCertInUserAndSystemRootCertStores
+
+            using (var microsoftDotCom = new X509Certificate2(TestData.MicrosoftDotComSslCertBytes))
+            using (var chainHolder = new ChainHolder())
+            {
+                X509Chain chainValidator = chainHolder.Chain;
+                bool chainBuildResult = chainValidator.Build(microsoftDotCom);
+
+                StringBuilder builder = new StringBuilder();
+                foreach (var status in chainValidator.ChainStatus)
+                {
+                    builder.AppendFormat("{0} {1}{2}", status.Status, status.StatusInformation, Environment.NewLine);
+                }
+
+                Assert.True(chainBuildResult, "Chain build for microsoft.com cert failed. Reason: " + builder.ToString());
+            }
+        }
```

Comment:
Since it's relying on system trust it ideally wouldn't be inner loop; so perhaps it makes more sense as a precondition check (or a ConditionalFact-driver) for the other test.  You should go ahead and set the RevocationDate to a fixed value in both ChainPolicy objects, though; so the tests don't start failing when the cert expires.  (And may as well set RevocationMode to None, since CRL checking isn't relevant)